app/views/layouts/application.html.erb

```diff
@@ -6,11 +6,10 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= stylesheet_pack_tag  'application',  media: 'all',  'data-turbolinks-track': 'reload'  %>
```

Comment:
Seems duplicated. Are you sure we need both `stylesheet_link_tag` and `stylesheet_link_tag`?

Reply:
It's not duplicate, but you're right. We only need pack_tag since it makes sense to use everything with webpacker. `stylesheet_link_tag` is the standard Rails asset pipelines where it loads from assets/stylesheets. I've removed this include and also that folder now, so that there's only one true location for styles assets.

Reply:
sweet